course05/homework01/skeleton_ivo/string.h

```diff
@@ -0,0 +1,30 @@
+#include <string>
+#include <ostream>
+#include <istream>
+
+class String
+{
+public:
+	String();
+	String(std::string);
```

Comment:
This will cause extra unnecessary work (allocation, copy), you should pass it via const ref